deb/common/rules

```diff
@@ -33,7 +33,7 @@ override_dh_auto_build:
 
 	# Build the compose plugin
 	cd /go/src/github.com/docker/compose \
-	&& make VERSION=$(COMPOSE_VERSION) DESTDIR=/usr/libexec/docker/cli-plugins build
+	&& make GO_VERSION=$(GO_VERSION_COMPOSE) VERSION=$(COMPOSE_VERSION) DESTDIR=/usr/libexec/docker/cli-plugins build
```

Comment:
This probably won't work, because both `deb` and `rpm` build-tools don't / can't use docker; they're plain `make` targets (but may be running inside a docker container)

Reply:
Yep I saw that, I'll test with the dedicated bake command defined in Compose makefile and check if using a container build will do the work